[Modify] render texture (#1592)
cocos-creator/creator-docs: en/asset/render-texture.md

        const cameraComp = this.getComponent(Camera);
        cameraComp.targetTexture = renderTex;
        const pass = this.model.material.passes[0];
        // Set the SAMPLE_FROM_RT macro to true so that RT can be displayed correctly on each platform
        const defines = { SAMPLE_FROM_RT: true, ...pass.defines };
        const renderMat = new Material();
        renderMat.initialize({
            effectAsset: this.model.material.effectAsset,
            defines,
        });
        this.model.setMaterial(renderMat, 0);
        renderMat.setProperty('mainTexture', renderTex, 0);
    }
}
```

> For more __Render Texture__ examples, please see these [test cases](https://github.com/cocos-creator/test-cases-3d/tree/v3.1/assets/cases/rendertexture).

Reviewer: ```suggestion
For more __Render Texture__ examples, please review the [test cases](https://github.com/cocos-creator/test-cases-3d/tree/v3.1/assets/cases/rendertexture).
```
